arches/app/templates/views/components/widgets/resource-instance-select.htm

```diff
@@ -62,17 +63,23 @@
                 <div class='rr-table-row'>
                     <div class="rr-table-row-initial">
                         <div class='rr-table-column icon-column'>
-                            <button data-bind="click: function(){window.open($parent.resourceEditorUrl+ko.unwrap($data.resourceId))}, clickBubble: false">
+                            <button data-bind="click: function(){window.open($parent.resourceEditorUrl+ko.unwrap($data.resourceId))}, clickBubble: false,
+                                    attr: {'aria-label': $root.translations.editRelatedResource + ' ' + $data.resourceName()}
+                            ">
                                 <i class="fa fa-pencil"></i>
                             </button>
                         </div>
-                        <div class='rr-table-column icon-column'>
-                            <button data-bind="click: $parent.deleteRelationship, clickBubble: false">
+                        <div class='rr-table-column icon-column' aria-label="{% trans '' %}">
+                            <button data-bind="click: $parent.deleteRelationship, clickBubble: false,
+                                    attr: {'aria-label': $root.translations.delete + ' ' + $root.translations.relationshipTo + ' ' + $data.resourceName()}
```

Comment:
we shouldn't concatenate these translations.  We should instead make a function that manages the entire string.